courses/machine_learning/deepdive2/text_classification/solutions/custom_tf_hub_word_embedding.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<p>Maybe want to change this to </p><p>PROJECT = "your-gcp-project-here" # REPLACE WITH YOUR PROJECT NAME</p><p>REGION = "us-central1" # REPLACE WITH YOUR BUCKET REGION e.g. us-central1</p><p> </p><p>I only suggest because I think there is a plan to run a setup.py script that will search/replace this string with individual/student projects/buckets and I think this was the decided upon format. </p><p> </p><p>That being said, I don't think everyone has been following this convention so perhaps a moot point. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>